test/forms/mappings/MappingsSpec.scala

```diff
@@ -57,4 +59,70 @@ class MappingsSpec extends SpecBase with Mappings{
       result.apply("value").value.get shouldBe "foobar"
     }
   }
+
+  "localDate" must {
```

Comment:
Perhaps having a test with spaces in between numbers:
```

    "bind a date with spaces in between digits" in {
      val result = testForm.bind(Map(
        "date.day" -> " 0   3  ",
        "date.month" -> " 0 4",
        "date.year" -> "2 0 2 1 "))
      result.value.get shouldBe LocalDate.of(2021 ,4 ,3)
    }
```

Reply:
Doesn't this test cover this scenario? I can always add more spaces for the digits https://github.com/hmrc/penalties-appeals-frontend/pull/464/files#diff-72900861a3dd996bd11066241fc51b44474f22885daa2441978cde0a84ccf53fR80

Reply:
Yeah, didn't notice that (need more coffee!)